src/App.tsx

```diff
@@ -1,43 +1,55 @@
-import React, { useState, useEffect } from 'react';
+
+import React, {useState, useEffect} from 'react';
+import './App.css';
 import axios from 'axios';
-import { Switch, Route, Link } from 'react-router-dom';
+import {Switch, Route,Link} from 'react-router-dom';
 import SetupPage from './pages/SetupPage/SetupPage';
 import GeneratePage from './pages/GenerateWalletPage/Generate';
 import ExistingPage from './pages/ExistingWalletPage/Existing';
 import HomePage from './pages/HomePage/HomePage';
-import './App.css';
+
+
+const Props = {
+  setPrivateKey: 'string'
+}
+
 
 const etherscanAPI = process.env.REACT_APP_ETHERSCAN_API;
 
 function App() {
   const [account, setAccount] = useState([]);
-  const [privateKey, setPrivateKey] = useState('');
+
+  const [privateKey, setPrivateKey] = useState('')
+
 
   useEffect(() => {
-    (async () => {
-      const { data } = await axios.get(
-        `http://api.etherscan.io/api?module=account&action=txlist&address=${privateKey}&startblock=0&endblock=99999999&sort=asc&apikey=${etherscanAPI}`,
-      );
-      setAccount(data);
-    })();
-  }, [privateKey]);
+    axios.get(`http://api.etherscan.io/api?module=account&action=txlist&address=${privateKey}&startblock=0&endblock=99999999&sort=asc&apikey=${etherscanAPI}`)
+    .then(res => setAccount(res.data))
+  }, [privateKey])
+
 
   return (
     <Switch>
       <div className="App">
         <Link to="/">MurdaMask</Link>
-        <Route exact path="/" render={() => <SetupPage />} />
-        <Route exact path="/generate" render={() => <GeneratePage />} />
-        <Route
-          exact
-          path="/existing"
-          render={() => <ExistingPage setPrivateKey={setPrivateKey} />}
-        />
-        <Route
-          exact
-          path="/homepage"
-          render={() => <HomePage privateKey={privateKey} />}
-        />
+
+        <Route exact path='/' render={() =>
+        <SetupPage />
+        }>
+        </Route>
+        <Route exact path='/generate' render={() => 
+        <GeneratePage />
+        }>
+        </Route>
+         <Route exact path='/existing' render={() => 
+        <ExistingPage setPrivateKey={setPrivateKey}/>
+        }>
+         </Route>
+         <Route exact path="/homepage" render={() => 
+           <HomePage privateKey={privateKey}/>
+         }>
+         </Route>
+
```

Comment:
Does not look there are any changes here either. Just formatting changes. you can ignore these changes too